src/rate_limiter.cc

```diff
@@ -178,10 +186,14 @@ RateLimiter::DequeuePayload(
     std::shared_ptr<Payload>* payload)
 {
   payload->reset();
-  if (payload_queues_.find(instances[0]->Model()) == payload_queues_.end()) {
-    LOG_INFO << "Should not print this ";
+  PayloadQueue* payload_queue;
```

Comment:
You may want to initialize this immediately to a nullptr to avoid unexpected behavior (e.g. segfault) if the code changes later.

Reply:
Update commit: [Init rate limiter ptr to null once declared](https://github.com/triton-inference-server/core/pull/165/commits/15781d6d2a08c7053bb9fa0b0ec304bb3de7fb6e)